.github/workflows/docker-tagged.yml

```diff
@@ -0,0 +1,42 @@
+name: Docker Tagged Build
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Get TAG
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+
+      - name: Login to GHCR
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          registry: ghcr.io
+
+      - name: Build image and push to container registies
```

Comment:
Let's make it multiarch at the same time? See how Akos did faas-netes

Reply:
I've got a separate PR ready to post for that, I wasn't sure how focused the team likes PRs to be - but I'm super happy to bring into this if you are.

Reply:
One PR is great

Reply:
It is done :)
checked it pulls as a multi-arch to my ARM64 pi-cluster